docker/DockerImages.md

```diff
@@ -0,0 +1,25 @@
+# Docker Images #
+
+The following docker images have been verified to successfully build the source-build repo.
```

Comment:
> have been

In the context of putting the doc in the repo, I think it's important to say how/when.

Reply:
I added how in the comments and a "Verification Date" column in the table.